<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_PlayerPreferences</class>
 <widget class="QWidget" name="GUI_PlayerPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>586</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labLogger">
       <property name="text">
        <string notr="true">Logger</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Gui::ComboBox" name="cbLogger"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cbShowTrayIcon">
     <property name="text">
      <string>Show system tray icon</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="cbUpdateNotifications">
     <property name="text">
      <string>Update notifications</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cbStartInTray">
     <property name="text">
      <string>Start hidden</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QWidget" name="widgetWarning" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labWarningHeader">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labWarning">
        <property name="styleSheet">
         <string notr="true">color: red;</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="cbCloseToTray">
     <property name="text">
      <string>Hide instead of close</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="cbInhibit">
     <property name="text">
      <string>Inhibit shutdown/suspend during playback</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::ComboBox</class>
   <extends>QComboBox</extends>
   <header>Gui/Utils/Widgets/ComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbShowTrayIcon</tabstop>
  <tabstop>cbStartInTray</tabstop>
  <tabstop>cbCloseToTray</tabstop>
  <tabstop>cbUpdateNotifications</tabstop>
  <tabstop>cbLogger</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
